cernopendata/modules/pages/views.py

```diff
@@ -86,6 +86,79 @@ def index():
                            records=results.hits.hits)
 
 
+@blueprint.route('/md/debug', methods=['HEAD', 'GET'])
+def md_debug():
+    """Test Markdown rendering of a page.
+
+    Route for quicker previewing markdown during development.
+    During edit replace contents of `cernopendata/static/test_data/debug.md`
+    and just refresh the view.
+    I.e. there is no need to re-populate db every time a change is made.
+
+    Command `cernopendata collect -v` has to be run everytime
+    there is a change in static resources.
+
+    """
+    f = open('cernopendata/static/test_data/debug.md', 'r')
+    # return render_template_string(
+    #     u"{{ text|markdown }}", text=f.read().decode("utf-8"))
+    return render_template('cernopendata_pages/md_template.html',
+                           content=f.read().decode("utf-8"))
+
+
+@blueprint.route('/education')
+@blueprint.route('/education/<string:experiment>')
+@register_breadcrumb(blueprint, '.education.experiment',
+                     lazy_title('%(experiment)s', 'experiment'),
+                     endpoint_arguments_constructor=lambda: {
+                         'experiment': request.view_args['experiment']})
+def education(experiment=None):
```

Comment:
remove these blueprints and the whole function as we are taking care of them in the faceted search.

Reply:
Accidentally included, sorry.